charts/backstage/values.yaml

```diff
@@ -120,6 +120,20 @@ backstage:
   extraEnvVarsSecrets:
   extraVolumeMounts: []
   extraVolumes: []
+  ## backstage resource requests and limits
+  ## ref: https://kubernetes.io/docs/user-guide/compute-resources/
+  ## @param resources.limits.memory The resources limits for the backstage containers memory
+  ## @param resources.limits.cpu The resources limits for the backstage containers cpu
+  ## @param resources.requests.memory The requested memory for the backstage containers
+  ## @param resources.requests.cpu The requested cpu for the backstage containers
+  ##
+  # resources:
```

Comment:
I think this is not recognized as an existing value unless you actually define it. Can you change this to `resources: {}` as the default value?